packages/inter-protocol/test/smartWallet/contexts.js

```diff
@@ -1,6 +1,8 @@
 import { BridgeId, deeplyFulfilledObject } from '@agoric/internal';
-import { unsafeMakeBundleCache } from '@agoric/swingset-vat/tools/bundleTool.js';
 import { makeStorageNodeChild } from '@agoric/internal/src/lib-chainStorage.js';
+// eslint-disable-next-line no-unused-vars -- used by TS
+import { coalesceUpdates } from '@agoric/smart-wallet/src/utils.js';
```

Comment:
something like `@typedef { Awaited<ReturnType<import('@agoric/smart-wallet/src/utils.js').coalesceUpdates>> } WalletState` might avoid increasing the bundle size... oh... this is a test, so there's no relevant bundle. NVM.